NewHorizons/Schemas/body_schema.json

```diff
@@ -2047,13 +2047,11 @@
       "description": "",
       "x-enumNames": [
         "Sockets",
-        "States",
-        "FailedValidation"
+        "States"
       ],
       "enum": [
         "sockets",
-        "states",
-        "FailedValidation"
+        "states"
```

Comment:
dupe again

Reply:
Not duplicated, it was just the dif rendering that it was removed and re-added